libtcmu.h

```diff
@@ -32,6 +32,11 @@ extern "C" {
 #include "libtcmu_common.h"
 #include "libtcmu_log.h"
 
+#define TCMU_CAPACITY_CHANGED   0x01
+#define TCMU_CONFIG_CHANGING	0x02
+#define TCMU_CONFIG_CHANGED	0x04
+#define TCMU_CAPACITY_CHANGE_FOR_CONFIG_CHANGE	0x08
```

Comment:
Let's rename to make it more clear what they are for. Maybe something like TCMU_DEV_STATE_something_sommething